### DISCUSSION TITLE
How do I customise an existing CSS inside custom.html (when using the template as Hugo module) ?

### DISCUSSION THREAD

**Discussion by andreagrandi** [CaiJimmy/hugo-theme-stack#953] [General]
Hello, I'm trying to customise the `.article-content code` styling.

I'm currently using `hugo-them-stack` as a Go module but before going through this path, I had simply cloned the whole theme and changed the scss in place and it was working as expected.

Now, using the module, I'm aware that (I'm following instructions here https://stack.jimmycai.com/config/header-footer ) I need to create a file named `layouts/partials/head/custom.html` and insert the CSS customisation there.

I inserted this:

```html
<style>
    /* Overwrite CSS variable */
    .article-content code {
        border-radius: var(--tag-border-radius);
        font-family: var(--code-font-family);
        color: #c25;
        padding: 1px 3px;
        background-color: #f7f7f9;
        border: 1px solid #e1e1e8;
        border-radius: 3px;
    }
</style>
```

but the style is not applied. 

Again, **when I was not using the module**, I **manually changed the style** here https://github.com/CaiJimmy/hugo-theme-stack/blob/03ee3369e1bfe794b5233c4661e66230a5944d69/assets/scss/partials/layout/article.scss#L263 and **everything was working as expected**.

<img width="724" alt="Screenshot 2024-02-21 at 16 53 15" src="https://github.com/CaiJimmy/hugo-theme-stack/assets/636391/626ffe06-f4e0-49e7-a8f2-b6e5f8663a26">

I suspect that the "selector" (is it called in this way? I'm really new to this stuff) I'm using (`.article-content code`) may be wrong or have a wrong syntax.

Why am I unable to apply the custom CSS?

p.s: the file seems to be detected by Hugo:
```shell
Change detected, rebuilding site.
2024-02-22 09:21:06.174 +0100
Template added WRITE         "/Users/andrea/Projects/andreagrandi-hugo/layouts/partials/head/custom.html"
```

What am I missing? Thanks for your help 🙏🏻

**Comment by CaiJimmy** [comment]
I'm not sure what's wrong either. I'm not able to reproduce it. 

I have created the same file with content, and the style has changed as expected:

![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/5889006/fa2ad68d-0fee-4b1f-adf1-e0704a228b2d)

![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/5889006/0ed9b108-f0c5-4376-8b7d-96cbcc0b298b)

Can you create a minimal reproducible example and upload it to GitHub (using the starter template)?

**Reply by andreagrandi** [reply]
Hi and thanks for your reply! I forgot to update this discussion. It looks like it was some cache issue. Hugo wasn't reloading the content properly and I kept seeing the old style. I rebuilt the pages and it worked, it's also live on my blog https://www.andreagrandi.it/posts/how-to-show-cover-image-pelican-blog-posts/
